app/controllers/webauth_controller.rb

```diff
@@ -18,10 +18,10 @@ def login
   # TODO: can't think of a reasonable way to do strong params here
 
   def login_file
-    redirect_to file_path(params.symbolize_keys)
+    redirect_to file_path(params.to_unsafe_h.symbolize_keys)
   end
 
   def login_iiif
-    redirect_to iiif_path(params.symbolize_keys)
+    redirect_to iiif_path(params.to_unsafe_h.symbolize_keys)
```

Comment:
Is this because there are no strong params set in this controller?  If not, is there a reason we would use `to_unsave_h` over `to_h`?  (I'm a bit naive about this change in rails 5).